scripts/sidebar-lib.js

```diff
@@ -73,9 +73,9 @@ async function generateSidebar(dir, opts = {}) {
 
   for (const fullPath of fullPaths) {
     if (fs.statSync(fullPath).isDirectory()) {
-      // process subdirectories recusively (back button is top level only!)
```

Comment:
I had already called this out explicitly, and yet somehow forgot to omit it in the opts the first time around.